check-node-version.js

```diff
@@ -0,0 +1,32 @@
+#! /usr/bin/env node
+
+const minimumVersion = '12.0.0';
+const currentVersion = process.version.replace('v', '');
+
+const minimumVersionMajor = parseInt(minimumVersion.split('.')[0]);
+const currentVersionMajor = parseInt(currentVersion.split('.')[0]);
+
+const usesMinimumVersion = currentVersionMajor >= minimumVersionMajor;
+
+if (usesMinimumVersion) {
+  process.exit();
+}
+
+console.log(
+  '' +
+    'Error: You are using Node.js version ' +
+    currentVersion +
+    ', but you need ' +
+    'Node.js version ' +
+    minimumVersion +
+    ' or higher to build and/or test axe-core.' +
+    '\n\n' +
+    'Install Node.js version ' +
+    minimumVersion +
+    ' or higher and try again.' +
+    '\n\n' +
+    'You can use nvm (https://github.com/creationix/nvm) to update your Node.js version.' +
```

Comment:
We ought to create an [`.nvmrc` file](https://github.com/nvm-sh/nvm/blob/master/README.md#nvmrc), so users can just `cd axe-core; nvm use`.

Would you mind creating one?

Reply:
Nope, it'd make it Just Work™

Reply:
But wouldn't that force them to go back to using node 12 instead of a later version like 16 if that's their default?

Reply:
Looking back, I realize the implications this could have if someone does have an automatic script.

With an .nvmrc, you could warn if someone is not using 12.0.0. But in doing so, we are implying that 12.0.0 should be used for development (instead of at least/minimum 12.0.0).